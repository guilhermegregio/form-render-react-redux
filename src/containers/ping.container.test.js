import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';

import PingContainer from './ping.container';
import configureStore from '../store/store';

const store = configureStore();

it('default render ping container', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Provider store={store}><PingContainer /></Provider>, div);

  expect(div).toMatchSnapshot();
});
