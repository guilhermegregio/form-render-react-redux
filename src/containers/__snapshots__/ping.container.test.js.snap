// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render ping container 1`] = `
<div>
  <div
    data-reactroot=""
  >
    <h1>
      Ping container
    </h1>
    <div>
      <h2>
        <!-- react-text: 5 -->
        is pinging: 
        <!-- /react-text -->
        <!-- react-text: 6 -->
        false
        <!-- /react-text -->
      </h2>
      <button>
        Start PING
      </button>
      <p>
        Teste
      </p>
    </div>
  </div>
</div>
`;
